packages/orbit-components/src/Alert/index.jsx

```diff
@@ -194,16 +199,19 @@ Title.defaultProps = {
 };
 
 const Content = styled(StyledDiv)`
-  display: block;
-  width: ${({ inlineActions }) => !inlineActions && "100%"};
+  ${({ inlineActions }) => css`
+    display: block;
+    width: ${!inlineActions && "100%"};
```

Comment:
It doesn't look like the change for mobile buttons has been implemented, they don't span full width.

Reply:
buttons on mobile should be handled by users. Because buttons passed as children :)